modules/javafx.controls/src/test/java/test/javafx/scene/control/skin/SkinCleanupTest.java

```diff
@@ -58,6 +60,77 @@
     private Stage stage;
     private Pane root;
 
+  //---------------- TreeView
+
+    /**
+     * Sanity: replacing the root has no side-effect, listener to rootProperty
+     * is registered with skin api
+     */
+    @Test
+    public void testTreeViewSetRoot() {
+        TreeView<String> listView = new TreeView<>(createRoot());
```

Comment:
minor: Please rename the variable to `treeView` in this method and others.

Reply:
darn c&p - thanks :)